src/tutorial/index.rst

```diff
@@ -209,7 +209,7 @@ In order to create a :class:`MessageSpatial2D::Description<flamegpu::MessageSpat
 
 .. note::
   
-    The Python interface does not support C++'s templates and nested classes so there are differences in naming style. In almost all cases the template argument is simply appended to the name.
+    The Python interface does not support C++'s templates and nested classes so there are differences in naming style. :ref:`In almost all cases the template argument is simply appended to the name.<Device Agent Communication>`.
```

Comment:
I don't understand this link, it doesn't seem relevant.

Reply:
Used the wrong link. Corrected it to `Pure Python Types` which has a table of suffix for functions.